src/components/ObsDetails/PhotoContainer.js

```diff
@@ -71,14 +71,15 @@ const PhotoContainer = ( { photo, onPress, style }: Props ): Node => {
       }}
     />
   );
-  const loadingIndicator = (
-    <ActivityIndicator
-      className={classnames(
-        "absolute",
-        loadSuccess !== null && "hidden"
```

Comment:
This indicator's animation was always running in the background and keeping the e2e test from idling and proceeding to the next test step.

Reply:
I don't know if there is an advantage. It is much more common to find the `{renderFunction()}` approach in RN documentation and teaching materials.